README.md

```diff
@@ -43,41 +44,130 @@ Check the [Troubleshooting section](#troubleshooting) if something goes wrong wi
+
+* Kubernetes Engine API and billing enabled for your project (see [here](https://cloud.google.com/kubernetes-engine/docs/quickstart)).
+
+In order to deploy polkadot on GCP you can use a the preset configuration file: ```create.remote.sample-GCP.json``` and issue the following command:  
+
+ ```node . create --config config/create.remote.sample-GCP.json --verbose```  
+	
+The process will start creating an instance of polkadot on GCP.
+By default a new cluster will be created with the name polkadot-deployer at your default location with 2 `n1-standard-2` nodes under the specified project ID. 
+
+If you wish to delete your remote deployment of polkadot, you can use the destroy [name] command:  
 
-1. Download the latest polkadot deployer from git issuing the following command:  
-```git clone git@github.com:w3f/polkadot-deployer.git``` and run ```npm install```to install all requirements.
-   
-2. In order to deploy polkadot using a the preset configuration file: my-gcp-testnet.json and issue the following command:  
-	```node . create --config config/create.remote.sample.json --verbose```
-	The process will start creating an instance of polkadot using GCP. A second cup of coffee is recommended at this point. 
+ ```node . destroy gcp-testnet```
+</p>
+</details>
 
-3. If you wish to delete your remote deployment of polkadot, you can use the destroy [name] command:  
- ```node . destroy testnet9```    
-  
-  More information on the polkadot-deployer usage commands can be found in the [usage](#usage) section.
+<details><summary>AWS</summary>
+<p>
+To make a deployment on AWS you're required to configure your AWS credentials. It's recommended to do so using the corresponding `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` and `AWS_DEFAULT_REGION` environment variables. You can set the required values for these variables following the provided [documentation](https://docs.aws.amazon.com/amazonswf/latest/awsrbflowguide/set-up-creds.html).
+
+In order to deploy polkadot on AWS you can use a the preset configuration file: ```create.remote.sample-AWS.json``` and issue the following command: 
+ 
+ ```node . create --config config/create.remote.sample-AWS.json --verbose``` 
+ 
+The process will start creating an instance of polkadot on AWS. The process with create a 2 node cluster using `m4.large` machines. An IAM role and a VPC will be created that will contain the Amazon EKS for the deployment along with the required security groups and ingress rules. You may review the entire process [here](https://github.com/w3f/polkadot-deployer/tree/master/terraform/aws).
+
+If you wish to delete your remote deployment of polkadot, you can use the destroy [name] command:  
+
+ ```node . destroy aws-testnet```  
+</p>
+</details>
+
+<details><summary>Azure</summary>
+<p>
+To deploy polkadot on Azure you're required to set your credentials. You can do this by following this 
```

Comment:
We need to describe the environment variables that the tool expects, the same as for the other providers. In this case they are `ARM_CLIENT_ID`, `ARM_CLIENT_SECRET`, `ARM_SUBSCRIPTION_ID`, `ARM_TENANT_ID` , `TF_VAR_client_id` and `TF_VAR_client_secret`

Reply:
trying to find a good documentation from ms that describes how to get all of them.